src/styles/filter.scss

```diff
@@ -0,0 +1,22 @@
+.filter {
+  display: flex;
+
+  &__link {
+    margin: 3px;
+    padding: 3px 7px;
+
+    color: inherit;
+    text-decoration: none;
+
+    border: 1px solid transparent;
+    border-radius: 3px;
+
+    &:hover {
+      border-color: rgba(175, 47, 47, 0.1);
```

Comment:
1) Use hex instead rgb
2) Better to use css variables for color

Check other cases